api/services/mapping/converters.js

```diff
@@ -219,45 +220,42 @@ const c = {
   // There is 2 source format
   // - From logstash
   // - From
-  toSearchDocument: (source) => {
```

Comment:
This method has been removed since. Please rebase to make sure there is no merge issue

Reply:
Yes I haven't had the chance to rebase yet.